lib/dgram.js

```diff
@@ -97,19 +99,33 @@ function newHandle(type, lookup) {
   throw new ERR_SOCKET_BAD_TYPE();
 }
 
+// Wrap "validateInt32" to make sure it won't throw.
+function isPossibleValidFD(fd) {
+  let valid = true;
+  try {
+    validateInt32(fd, 'fd', 0);
```

Comment:
Please use `isInt32` instead. That does exactly what is necessary here.

Reply:
Thanks for your advice. As we need to make sure that the `fd` is greater than `0`, I use `isUint32` directly. 

Another test is added that it should return `errno` when the type of fd is not `"UDP"`.